prow/cmd/deck/static/common/rerun.ts

```diff
@@ -0,0 +1,151 @@
+          <i class='rerunModal-openOut material-icons state triggered' style='color: gray'>open_in_new</i>
+          </a>)
+        </div>
+      </div>
+      <div class="rerunModal-accordion">
+        <button class="rerunModal-accordionButton">
+          <i class='rerunModal-expandIcon material-icons state triggered' style='color: gray'>expand_more</i>
+          kubectl command
+        </button>
+        <div class="rerunModal-accordionPanel">
+          <div class="accordion-panel-content">
+            <p class="rerunModal-commandDescription">${getCommandDescription(LATEST)}</p>
+            <div class="rerunModal-commandContent">
+              <div class="rerunModal-command">${getCommand(commandURL)}</div>
+              <a class="rerunModal-copyButton mdl-button mdl-js-button mdl-button--icon">
+                <i class='material-icons state triggered' style='color: gray'>content_copy</i>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    `;
+
+    const rerunLatestOption = rerunElement.querySelector('#rerunLatestOption');
+    const rerunCommand = rerunElement.querySelector('.rerunModal-command');
+    const rerunCommandDescription = rerunElement.querySelector('.rerunModal-commandDescription');
+    rerunLatestOption.addEventListener('click', () => {
+      commandURL = latestURL;
+      rerunCommandDescription.innerHTML = getCommandDescription(LATEST);
+      rerunCommand.innerHTML = getCommand(commandURL);
+    });
+    const rerunOriginalOption = rerunElement.querySelector('#rerunOriginalOption');
+    rerunOriginalOption.addEventListener('click', () => {
+      commandURL = originalURL;
+      rerunCommandDescription.innerHTML = getCommandDescription(ORIGINAL);
+      rerunCommand.innerHTML = getCommand(commandURL);
+    });
+    const rerunCopyButton = rerunElement.querySelector('.rerunModal-copyButton');
+    rerunCopyButton.addEventListener('click', () => {
+      copyToClipboardWithToast(getCommand(commandURL));
+    })
+    const accordion = rerunElement.querySelector('.rerunModal-accordionButton');
+    const accordionPanel = rerunElement.querySelector('.rerunModal-accordionPanel');
+    const expandIcon = rerunElement.querySelector('.rerunModal-expandIcon');
+    accordion.addEventListener('click', () => {
+      if (!accordionPanel.classList.contains('rerunModal-accordionPanel--expanded')) {
+        accordionPanel.classList.add('rerunModal-accordionPanel--expanded');
+        expandIcon.classList.add('rerunModal-expandIcon--expanded');
+      } else {
+        accordionPanel.classList.remove('rerunModal-accordionPanel--expanded');
+        expandIcon.classList.remove('rerunModal-expandIcon--expanded');
+      }
+    });
+
+    if (rerunCreatesJob) {
+      const runButton = document.createElement('a');
+      runButton.innerHTML = "<button class='mdl-button mdl-js-button mdl-button--raised mdl-button--colored'>Rerun</button>";
+      runButton.onclick = async () => {
```

Comment:
See previous note about try/catch on the below async/await logic.

Reply:
Updated!